pkgs/by-name/sc/scion/package.nix

```diff
@@ -1,5 +1,5 @@
 { lib
-, buildGoModule
+, buildGo121Module
```

Comment:
this should be done in all-packages.nix

https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults

Reply:
@kirillrdy scion is no longer using buildGoModule121, this was only temporary. The issue was fixed upstream.